File: 1011/1011.cs
using System; 
class URI {
    static void Main(string[] args) { 
        double r, v;

        r = double.Parse(Console.ReadLine());
        v = (4 / 3.0) * 3.14159 * r * r * r;

        Console.WriteLine("VOLUME = " + v.ToString("F3"));
    }
}

File: 1011/1011.rs
use std::io;

fn main() {
    let mut input = String::new();
    io::stdin().read_line(&mut input);
    let r: f64 = input.trim().parse().unwrap();

    println!("VOLUME = {:.3}", (4.0 / 3.0) * 3.14159 * r * r * r);
}